usr/share/laptop-mode-tools/locale/zh_CN.ts

```diff
@@ -3,413 +3,413 @@
 <context>
     <name>MainWidget</name>
     <message>
-        <location filename="../gui/lmt.py" line="55"/>
+        <location filename="../../../../gui/lmt.py" line="53"/>
```

Comment:
Is this path correct ?

I assume the CWDIR would be ` usr/share/laptop-mode-tools/locale/` and in your proposed patch, if you traverse "../../../../" you are essentially chdir()ing to `/`

Can you please enlighten on this ?